applications/vanilla/controllers/api/DiscussionsApiController.php

```diff
@@ -307,6 +307,58 @@ public function normalizeOutput(array $dbRecord, $expand = []) {
         return $result;
     }
 
+    /**
+     * Get a discussion's quote data.
+     *
+     * @param int $id The ID of the discussion.
+     *
+     * @return array The discussion quote data.
+     *
+     * @throws NotFoundException If the record with the given ID can't be found.
+     * @throws \Exception if no session is available.
+     * @throws \Vanilla\Exception\PermissionException if the user does not have the specified permission(s).
+     * @throws \Garden\Schema\ValidationException If the output schema is configured incorrectly.
+     */
+    public function get_quote($id) {
```

Comment:
Same kinda comments as I made on the comment quote method (permission, schema type, etc.).